src/main/java/org/galatea/starter/entrypoint/PriceFinderRestController.java

```diff
@@ -0,0 +1,46 @@
+package org.galatea.starter.entrypoint;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import net.sf.aspect4log.Log;
+import net.sf.aspect4log.Log.Level;
+import org.galatea.starter.domain.AVStock;
+import org.galatea.starter.service.PriceFinderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+//sample call: 'http://localhost:8080/pricefinder?text=IBM&n=10'
+
+/**
+ * REST Controller that listens to http endpoints and allows the caller to send text to be
+ * processed.
+ * Should process GET requests for data regarding stock ticker
+ */
+@RequiredArgsConstructor
+@Slf4j
+@Log(enterLevel = Level.INFO, exitLevel = Level.INFO)
+@RestController
+public class PriceFinderRestController extends BaseRestController {
+
+
+  @Autowired
```

Comment:
Remove ```@Autowired```, replace with constructor injection + lombok.
You already have ```@RequiredArgsConstructor```, which will create a constructor automatically that initializes every ```final``` field. Spring will automatically autowire your ```PriceFinderService``` because there is only one constructor.